docs/assets/js/docs.js

```diff
@@ -1,16 +1,22 @@
-m_triggers = document.querySelectorAll("[data-trigger-modal]");
+/* Modals */
+const modalTriggers = document.querySelectorAll("[data-trigger-modal]");
 
-for(var i=0; i < m_triggers.length; i++) {
-  mt = m_triggers[i];
+for (const i = 0; i < modalTriggers.length; i++) {
```

Comment:
This one needs to be a `let`, because it's reassigned each iteration of the loop.

Reply:
Fixed